spec/ddtrace/contrib/http/request_spec.rb

```diff
@@ -96,6 +96,7 @@
               expect(span).to be_a_kind_of(Datadog::Span)
               expect(http).to be_a_kind_of(Net::HTTP)
               expect(request).to be_a_kind_of(Net::HTTP::Get)
+              expect(request).to be_a_kind_of(Net::HTTP::Get)
```

Comment:
Did we accidentally duplicate the line above here?

Reply:
not sure where that one came from, looks like a dupe to me too, removed